packages/api/src/lib/structures/api/CookieStore.ts

```diff
@@ -103,6 +103,29 @@ export class CookieStore extends Map<string, string> {
 		return entry;
 	}
 
+	/**
+	 * Parses a host using the {@link https://github.com/lupomontero/psl psl} library to extract the domain.
+	 * This is used for the domain of the cookie
+	 * @param host The hot to parse
+	 * @returns Either the host in all lower case or the parsed domain, ready for use on cookies
+	 */
+	private getHostDomain(host: string): string {
+		// Transform the host to lower case
```

Comment:
I guess you can mention the RFC to make kyra N.U.T.

Reply:
Yeah - RFC 6265 5.1.3 Domain Matching